docs/getting_started/navigating_docs.md

```diff
@@ -0,0 +1,29 @@
+# Navigating CURC Documentation
+
+CU Research Computing (CURC) offers a variety of services encompassing High Performance Computing (HPC), cloud computing assistance, and storage solutions. Many of these services are available to users associated with CU Boulder, Anschutz Medical Campus (AMC), Colorado State University (CSU), and institutions who are members of the Rocky Mountain Advanced Computing Consortium (RMACC). Our team understands that navigating the various services we provide can be overwhelming for both new and experienced users. To aid users, we have constructed flowcharts that outline the core areas of our documentation that every user on our system should become familiar with. Please refer to the section [High Performance Computing Flowchart](#high-performance-computing-flowchart) for a diagram on how to navigate our HPC systems (clusters) and storage solutions. For a diagram on the various cloud computing resources we can assist you with, please refer to the [Cloud Computing Flowchart](#cloud-computing-flowchart) section. 
+
+## High Performance Computing Flowchart
+
+At CURC we host two HPC clusters, Alpine and Blanca, in addition to storage solutions. To utilize these resources there are several items that are important to be aware of. Although our documentation covers a variety of other topics, below we provide a flowchart that we believe identifies the key areas users of these services should read. 
```

Comment:
```suggestion
At CURC, we host two HPC clusters, Alpine and Blanca, along with various storage solutions. To make the most of these resources, there are several crucial points to consider. While our documentation addresses many topics, we've included a flowchart below that highlights the key areas users should focus on.
```

Reply:
I modified this sentence slightly from what you provide, but kept a lot of the elements. Thank you for the suggestions!